main.js

```diff
@@ -0,0 +1,81 @@
+const bookList = document.querySelector('.bookList');
+const form = document.querySelector('.formInput');
+const [title, author] = form.elements;
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add an ID property to make it easier to remove duplicates values.